packages/hash/frontend/src/components/GlideGlid/utils/interactable-manager/utils.ts

```diff
@@ -7,6 +7,22 @@ export const drawArgsToPath = (args: DrawArgs<CustomCell>): CellPath => {
   return `${tableId}-${col}-${row}`;
 };
 
+export const splitPath = (path: CellPath) => {
+  const split = path.split("-");
+
+  if (split.length !== 3) {
+    throw new Error(`CellPath should have '{tableId}-{col}-{row}' format`);
+  }
+
+  const [tableId, col, row] = split;
+
+  return {
+    tableId: tableId!,
```

Comment:
I think this is a tiny bit clearer and avoids asserting
```suggestion
  const [tableId, col, row] = path.split("-");

  if (!tableId || !row || !col) {
    throw new Error(`CellPath should have '{tableId}-{col}-{row}' format`);
  }

  return {
    tableId,
```